static/js/hotkey.js

```diff
@@ -515,6 +515,11 @@ exports.process_hotkey = function (e, hotkey) {
         }
     }
 
+    // Prevents up and down navigation along the message list when lightbox is open/
+    if ((event_name === 'up_arrow' || event_name === 'down_arrow') && overlays.lightbox_open()) {
+        return true; // True - inorder to prevent the default action of browser as well.
```

Comment:
Maybe `e.preventDefault()` is more explicit? Not sure if it does the same as you're expecting with the return, though.